app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,7 @@
     <%= javascript_importmap_tags %>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= render 'layouts/analytics' %>
```

Comment:
The name of the file is a bit misleading as it isn't a layout. What about moving this to `shared` and going with a more obvious naming: `shared/_plausible_tracking` ?